Client/Assets/NoImageModeHelper.js

```diff
@@ -0,0 +1,72 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ + * License, v. 2.0. If a copy of the MPL was not distributed with this
+ + * file, You can obtain one at http:mozilla.org/MPL/2.0/. */
+
+(function() {
+  if (!window.__firefox__) {
+    window.__firefox__ = {};
+  }
+  var observers = [];
+  var isEnabled = false;
+
+  function initializeStyleSheet() {
```

Comment:
Nit: whitespace needed between function name and argument list.

`function initializeStyleSheet () {`